domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt

```diff
@@ -20,7 +20,10 @@ class StoryProgressTestHelper @Inject constructor(
     return Date().time - EIGHT_DAYS_IN_MS
   }
 
-  /** Creates a partial story progress for a particular profile. */
+  /** Creates a partial story progress for a particular profile.
+   * @param profileId The profile we are setting partial progress of the fraction story for.
+   * @param timestampOlderThanOneWeek If the timestamp for this topic progress is more than one week ago.
+   */
```

Comment:
```suggestion
  /**
   * Creates a partial story progress for a particular profile.
   *
   * @param profileId the profile we are setting partial progress of the fraction story for
   * @param timestampOlderThanOneWeek if the timestamp for this topic progress is more than one week ago
   */
```

Per https://developer.android.com/kotlin/style-guide#formatting_2.

Ditto elsewhere.

Reply:
Ack, done